libr/main/radiff2.c

```diff
@@ -1246,12 +1246,12 @@ R_API int r_main_radiff2(int argc, const char **argv) {
 			RDiff *d = r_diff_new ();
 			if (d) {
 				d->verbose = verbose;
-				if (mode == MODE_DIST_LEVENSTEIN) {
-					d->type = 'l';
-				} else if (mode == MODE_DIST_MYERS) {
+				if (mode == MODE_DIST_MYERS) {
```

Comment:
if levenstein is catched in https://github.com/radareorg/radare2/pull/16986/files#diff-da3b7af0a797b3cf25e710584fb60571R1244 it should be throwing a warning or something instead of using the default method instead.

Also i would like to undestand and maybe get documnetd about the differences of all the distance diffing algorithms and why an user may pick one or another, this should be clear in the manpage too.

Reply:
also the title is drop MODE_DIST: and mode_dist is still there.